.github/workflows/bashlib.sh

```diff
@@ -89,6 +89,8 @@ EOF
 setup-mysql() {
   say "::group::Initialize database"
   mysql -h 127.0.0.1 -P 13306 -u root --password=root <<-EOF
+    SET GLOBAL transaction_isolation='READ-COMMITTED';
```

Comment:
NOTE: this didn't work first try, so trying `SET GLOBAL TRANSATION` bellow instead...